forecasting/tests/unittests/models/common.py

```diff
@@ -0,0 +1,40 @@
+"""Common utils and data for all model tests"""
+import random
+
+import pandas as pd
+from gluonts.dataset.common import ListDataset
+
+DUMMY_DATASET = ListDataset(
```

Comment:
Thoughts on hosting a few small real datasets in S3 for testing? This is done in Tabular, Text, and Vision currently. We could also wait until a future PR if we think it would be too cumbersome to do so at this stage.

Reply:
added a todo